README.md

```diff
@@ -476,6 +476,36 @@ As with `Route` from `react-router-dom`, `<SecureRoute>` can take one of:
 
 `LoginCallback` handles the callback after the redirect to and back from the Okta-hosted login page. By default, it parses the tokens from the uri, stores them, then redirects to `/`. If a `SecureRoute` caused the redirect, then the callback redirects to the secured route. For more advanced cases, this component can be copied to your own source tree and modified as needed.
 
+#### errorComponent
+
+By default, LoginCallback will display any errors from `authState.error`.  If you wish to customise the display of such error messages, you can pass your own component as an `errorComponent` prop to `<LoginCallback>`.  The `authState.error` value will be passed to the `errorComponent` as the `error` prop.
+
+#### onAuthResume
+
+In special cases where an external auth (such as a social IDP) redirects back to your application AND your Okta sign-in policies require additional authentication factors before authentication is complete, the redirect to your application redirectUri callback will be an `interaction_required` error.  Such an error can be handled like any other error, or you can pass an `onAuthResume` function as a prop to `<LoginCallback>`.  `onAuthResume` is expected to handle the `interaction_required`.  Most commonly (for this not-common situation) you will want your application to resume the login.  If using the Okta SignIn Widget, redirecting to your login route will allow the widget to automatically resume your authentication transaction.
```

Comment:
I would not describe Social IDP as either "special" or "not common", although it is a new feature for this SDK.

I'm not sure what "handled like any other error" means; let's focus on the fact that this is the signal for your app to **resume the auth flow**. 

When mentioning the widget, let's link to that repo and also mention that you must set the `useInteractionCodeFlow` option

Reply:
This is more than Social IDP, this is Social IDP _plus_ and an additional factor (otherwise the redirection to the redirectUri won't come back as `interaction_required`.  That's uncommon as far as I know, and I was trying to highlight that so most devs know if they need to even consider this feature.

> I'm not sure what "handled like any other error" means

That is me trying to find text that will work both before and after the LoginCallback errorComponent is fixed, as well as describe the default behavior if you get `interaction_required` and haven't provided an `onAuthResume` function.  

> also mention that you must set the useInteractionCodeFlow option

I didn't want to mention an as-of-yet undocumented option used on a not-yet-generally-available feature.  

I'll take a stab at addressing the points you raise here, but the above can cause some confusion.